internal/namespaces/rdb/v1/custom_instance_connect.go

```diff
@@ -0,0 +1,202 @@
+package rdb
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"reflect"
+	"runtime"
+	"strings"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+	"github.com/scaleway/scaleway-sdk-go/api/rdb/v1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+type instanceConnectArgs struct {
+	Region     scw.Region
+	InstanceID string
+	Username   string
+	Database   *string
+	Tool       *string
+}
+
+type engineFamily string
+
+const (
+	Unknown        = engineFamily("Unknown")
+	PostgreSQL     = engineFamily("PostgreSQL")
```

Comment:
Should it exist in rdb API ?

Reply:
@nlm What do you think about having const in your API about the kind of engine a given instance is running? Without the version, ie PostgreSQL instead of PostgreSQL-12.